oauth2client/contrib/flask_util.py

```diff
@@ -443,7 +443,12 @@ def credentials(self):
 
     def has_credentials(self):
         """Returns True if there are valid credentials for the current user."""
-        return self.credentials and not self.credentials.invalid
```

Comment:
@nathanielmanistaatgoogle Is there any way to do a survey of how and if people are using features like this?